pkg/node/manager.go

```diff
@@ -41,8 +41,11 @@ const (
 )
 
 var (
-	mutex lock.RWMutex
-	nodes = map[Identity]*Node{}
+	mutex                 lock.RWMutex
+	nodes                 = map[Identity]*Node{}
+	ciliumHostInitialized = false
+	usePerNodeRoutes      = false
+	auxPrefixes           = []*net.IPNet{}
```

Comment:
Since these are package variables (which is probably unintentional) and file variables, could you comment what/when/how they're to be used. In particular, what is mutex protecting?

Reply:
fixed